python/lsst/analysis/drp/dataSelectors.py

```diff
@@ -320,7 +360,7 @@ def __call__(self, df, **kwargs):
         """
 
         result = None
-        flagCols = ["_psfFlux_flag", "_pixelFlags_saturatedCenter", "_extendedness_flag"]
+        flagCols = ["psfFlux_flag", "pixelFlags_saturatedCenter", "extendedness_flag"]
         filterColumns = ["xy_flag"]
         filterColumns += [band + flag for flag in flagCols for band in self.bands]
```

Comment:
Strongly suggest doing the `_` string parsing here, and not defining anything with extra underscores (bands or flags). So it would be `band + '_' + flag for ...`

This happens in a few places, be sure to catch all of them.

Reply:
I understand why you did it, but I'm concerned it will introduce bugs when clueless people do things like add a band and don't magically know that band names in this context should have trailing underscores. I think adding `+ '_'` each time you build a column list is a lot less inconvenient.

Reply:
Per discussion, some columns have `_` and some don't, how fun. We came up with a slightly less awful standardization scheme along the lines of `filterColumns += [band + '_' + flag if len(band) > 0 else band + flag for flag in flagCols for band in self.bands]`.